db/migrate/20230808110225_create_recipes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRecipes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipes do |t|
+      t.string :name
+      t.integer :preparation_time
+      t.integer :cooking_time
+      t.string :description
+      t.boolean :public
+      t.references :users, null: false, foreign_key: true
```

Comment:
Instead of having reference as :users , please ensure its singular :user , this will ensure proper that your tables in the schema file have an appropriate name on the reference column. ie `user_id` instead of the wrong column name `users_id`